packages/manager/src/manager.ts

```diff
@@ -99,6 +115,7 @@ export class Manager {
 
     const threeId = await ThreeIdProvider.create(threeIdConfig)
     this.threeIdProviders[threeId.id] = await ThreeIdProvider.create(threeIdConfig)
+    void this.store.storeDID(threeId.id, this.threeIdProviders[threeId.id].keychain._keyring.seed)
```

Comment:
Any reason to use `void` here rather than `await` please?

Reply:
yeah didnt really have to await these, especially cache, but yeah actually should be up to the cache/store implementation instead, otherwise will end up with unexpected behavior in tests/etc, changed these